src/components/network/equipment-table-editors.js

```diff
@@ -86,8 +89,69 @@ export const NumericalField = ({
                     },
                     min: { min },
                     max: { max },
+                    step: 'any',
                 }}
             />
-        </Tooltip>
+        );
+    }
+
+    return (
+        <div>
+            {min !== undefined || max !== undefined ? (
+                <Tooltip
+                    title={intl.formatMessage({ id: 'MinMax' }, { min, max })}
+                >
+                    {renderNumericText()}
+                </Tooltip>
+            ) : (
+                renderNumericText()
+            )}
+        </div>
+    );
+};
+
+export const NameField = ({ defaultValue, setter, style, ...props }) => {
+    return (
+        <TextField
+            defaultValue={defaultValue}
+            onChange={(ev) => setter(ev.target.value)}
+            {...props}
+            size={'small'}
+            margin={'none'}
+            style={{ ...style, padding: 0 }}
+            inputProps={{
+                style: {
+                    textAlign: 'center',
+                    fontSize: 'small',
+                    flexGrow: 1,
```

Comment:
what is the flexGrow useful for ?

Reply:
it is useless. I copy/paste all the style from another TextField Editor to look like the same. I will remove it (in both)